solutions/observability/logs/logs-explorer.md

```diff
@@ -3,62 +3,51 @@ mapped_pages:
   - https://www.elastic.co/guide/en/observability/current/explore-logs.html
   - https://www.elastic.co/guide/en/serverless/current/observability-discover-and-explore-logs.html
 applies_to:
-  stack: all
-  serverless: all
+  stack: ga
+  serverless: ga
 ---
 
-# Logs Explorer [explore-logs]
+# Explore logs in Discover [explore-logs]
```

Comment:
Should we mention somewhere here that the logs contextual experience is only enabled when the solution navigation is set to Observability? cc @LucaWintergerst